### DISCUSSION TITLE
Quasar v2.4.13 Meta props are dynamically not changing

### DISCUSSION THREAD

**Discussion by GowthamReddyAnala** [quasarframework/quasar#12178] [CLI - SSR mode]
import { useMeta } from 'quasar';
import {  ref } from "vue";

	setup() {
		const title = ref('First page'); // we define the “title” prop
		const myDescription = ref({
			name: "description",
			content: "First page quasar app",
		});
		useMeta(() => {
			return {
			// whenever “title” from above changes, your meta will automatically update
				title: title.value,
				meta: {
				myKey: myDescription.value,
			},
		    };
	    });
	    function setMetaDescription(){
	            title.value = 'Title Changed';
		    myDescription.value = {
			    name: "description",
			    content: "Hey I am changed",
		    }
	    }
		return {
			setMetaDescription
		};
	},
	
	mounted(){
	      this.setMetaDescription()
	}
	
	
	

The result is the title meta prop is always updated, but the description meta prop is not updated

**Comment by rstoenescu** [comment]
Ran your code. Works as expected.

<img width="505" alt="image" src="https://user-images.githubusercontent.com/4665641/150928158-ae54fa3f-26c9-42c6-98b1-b84c9b856cc3.png">

**Reply by GowthamReddyAnala** [reply]
Hi,

I checked in the view page source it hasn't reflected


![ssrrender](https://user-images.githubusercontent.com/22943477/150946470-90252dbf-6d73-4b4b-b4bb-a54f7cddce56.PNG)

thank you

**Reply by yusufkandemir** [reply]
@GowthamReddyAnala view page source only displays the initial page source, it's not dynamic. If you look carefully you will see the title is also old, even though the tab title got updated.
If you use inspect feature of dev tools, you will see the updated results dynamically. You can access it by right-clicking anywhere inside the page then clicking "Inspect" or by using `Ctrl + Shift + C` shortcut.

**Reply by GowthamReddyAnala** [reply]
Hi 

```
import { useMeta } from "quasar";
import { ref } from "vue";
export default {
	setup() {
		const title = ref("First page"); 
		const myDescription = ref({
			name: "description",
			content: "First page quasar app",
		});
		useMeta(() => {
			return {
				title: title,
				titleTemplate: (title) => `${title.value}`,
				meta: {
					description: myDescription.value,
				},
			};
		});
		const setMetaContent = () => {
			title.value = "Title Changed";
			myDescription.value = {
				name: "description",
				content: "Hey I am changed",
			};
		}
		return {
			setMetaContent,
		};
	},
	serverPrefetch() {
		return this.setMetaContent();
	},
	mounted() {
		this.setMetaContent();
	},
};
```


Above example Title updating in view page source but description not getting update

![image](https://user-images.githubusercontent.com/22943477/152382835-975bb2ab-774c-48fb-9c51-85878a1b8279.png)

Please help me did I missing anything, I want to update description too
